install/kubernetes/helm/istio/values-istio-demo-grafana-config.yaml

```diff
@@ -0,0 +1,545 @@
+# Top level istio values file has the following sections.
```

Comment:
Please don't add this. It is not needed. When you do `helm install -f values-istio-demo-grafana-config.yaml` applies the default `values.yaml` first, then overrides things with  `values-istio-demo-grafana-config.yaml`. So you don't need to add all of this stuff, you just need the 1 or 2 things you change.

Even if you fix that though, it still should not be in this file, though they should be in `grafana/values.yaml`. 

If you want to show examples, like `email-smtp.eu-west-1.amazonaws.com:2587`, please make them a comment. Otherwise every Istio user is going to get this value which is going to break things.

Reply:
Hi John, thanks for the guidance on this. I have made the necessary changes as requested.